recipes/libsamplerate/all/conanfile.py

```diff
@@ -57,6 +57,10 @@ def _configure_cmake(self):
         self._cmake.configure()
         return self._cmake
 
+    def build_requirements(self):
+        if self.settings.os == "Macos" and tools.Version(self.version) >= "0.2.2":
```

Comment:
```suggestion
        if tools.is_apple_os(self.settings.os) and self.options.shared and tools.Version(self.version) >= "0.2.2":
```

I guess due to https://github.com/libsndfile/libsamplerate/commit/b81ed273be04b86855e45483d1f13ae46338c2de

Reply:
Thank you for your investigation.
I 100% agree your suggestion.